man/adoc/bpftrace.adoc

```diff
@@ -2391,9 +2391,9 @@ kfunc:kvm:x86_emulate_insn { @ = count(); }
 === kprobe and kretprobe
 
 .variants
-* `kprobe:fn`
-* `kprobe:fn+offset`
-* `kretprobe:fn`
+* `kprobe[:mod]:fn`
```

Comment:
nit: maybe just spell this out mod -> module

Reply:
I think that would look nicer, but I was copying what the `kfunc` documentation does above -- I'll change both.